github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/GitHubPRJobDslExtension.java

```diff
@@ -44,4 +46,18 @@ public Object commitStatusOnGH(Runnable closure) throws ANTLRException {
                 null
         );
     }
+
+    @DslExtensionMethod(context = StepContext.class)
+    public Object updateStatusOnGH(Runnable closure) {
+        GitHubPRStatusStepDslContext context = new GitHubPRStatusStepDslContext();
+        executeInContext(closure, context);
+
+        return new GitHubPRBuildStatusPublisher(
```

Comment:
Looks like the publisher already implements the `SimpleBuildStep` interface: https://github.com/KostyaSha/github-integration-plugin/blob/master/github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/publishers/GitHubPRAbstractPublisher.java#L41 . Does this mean I can use publishers in the build step?